### PR TITLE
Feature/domicile transfer duration

### PR DESCRIPTION
**Pull request by VilliGunn**
# Transfer duration screen

https://app.asana.com/0/1199733748699731/1199513406635474

## What

- Added initial screen for the Transfer duration
- `<DatePickerController />` now accepts `backgroundColor` prop
- Moved the Datepicker types to an external types file.
  - To be able to consume exported types both in `.ts` and `.tsx` files they seem to need to be exported from a `.ts` file
- Temporary solutions for now
  - Instead of 'sublabel' I added a tooltip
  - Instead of rendering the 'DatePicker' below the 'Temporary' option it is below both radio buttons

## Screenshots / Gifs

Screenshot 2021-01-13 at 15 59 53

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by Rikkiabb — DISMISSED** [review]
LGTM 🚀

File: libs/application/templates/children-domicile-transfer/src/forms/ChildrenDomicileTransferForm.ts [hunk 1]
```diff
@@ -103,22 +105,41 @@ export const ChildrenDomicileTransferForm: Form = buildForm({
...
       title: 'Gildistími',
       children: [
-        buildTextField({
-          id: 'children',
-          title: 'children',
+        buildMultiField({
+          id: 'duration',
+          title: 'Í hve langan tíma á samningurinn að gilda?',
+          description:
+            'Veldu í hversu langan tíma samningurinn á að gilda. Hægt er að gera tímabundna lögheimilisbreytingu til a.m.k. 6 mánaða eða lengur eða velja að samningur gildi til frambúðar.d',
```
Comment (oddsson) [inline libs/application/templates/children-domicile-transfer/src/forms/ChildrenDomicileTransferForm.ts:115]:
```suggestion
            'Veldu í hversu langan tíma samningurinn á að gilda. Hægt er að gera tímabundna lögheimilisbreytingu til a.m.k. 6 mánaða eða lengur eða velja að samningur gildi til frambúðar.',
```

**Review by oddsson — DISMISSED** [review]
Nice! There is a typo that needs fixing but other than that, good work!

**Review by baering — APPROVED** [review]
LGTM

**Review by oddsson — APPROVED** [review]

**Review by Rikkiabb — APPROVED** [review]

**Pull request merged**